### DISCUSSION TITLE
Combine history search only with Fig?

### DISCUSSION THREAD

**Discussion by Postmodum37** [marlonrichert/zsh-autocomplete#423] [Q&A] [answered]
Hey, I am testing [Fig](https://fig.io/) and was wondering if there is an easy way to just enable the history menu that opens up without the autocompletion part of this plugin? I've tried setting up the `zstyle ':autocomplete:*' min-input 200` but it only works initially when nothing is typed in a terminal.

**Comment by marlonrichert** [comment, marked as answer]
You can disable autocompletion by adding this to your `.zshrc` file _before_ sourcing Zsh Autocomplete: 
```
zstyle ':autocomplete:*' async no
```

**Reply by Postmodum37** [reply]
Unfortunately, it doesn't help. Autocompletion still happens and overrides Fig functionality 😔

**Reply by marlonrichert** [reply]
I forgot to mention: You need to declare that `zstyle` _before_ sourcing Zsh Autocomplete. It does nothing at runtime.

**Reply by Postmodum37** [reply]
Works like a charm, thank you! 👍
